app/build.gradle

```diff
@@ -120,6 +130,7 @@ dependencies {
   androidTestImplementation "com.android.support.test:rules:$versions.supportTest"
   androidTestImplementation "com.google.truth:truth:$versions.truth"
   androidTestImplementation "com.github.blocoio:faker:$versions.faker"
+  androidTestImplementation "android.arch.persistence.room:testing:$versions.room"
```

Comment:
This is only required for writing migration Android tests, but sure let's keep this